services/portal/GWAS/GWASTasks.js

```diff
@@ -15,6 +15,42 @@ module.exports = {
         I.saveScreenshot('GWAS_page.png');
     },
 
+    goToAnalysisPage() {
+        I.amOnPage(GWASUIAppProps.analysisPath);
+        I.wait(5);
+        I.saveScreenshot('analysis.png');
+    },
+
+    async selectTeamProject1() {
+        I.seeElement(GWASUIAppProps.TeamProjectSelectorBox);
+        I.click(GWASUIAppProps.TeamProjectSelectorDropdown);
+        I.click(GWASUIAppProps.TeamProject1Select);
+        I.saveScreenshot('GwasTeamSelect1.png');
+        I.click(GWASUIAppProps.TeamProjectSubmitButton);
+        I.seeElement('.team-project-header');
+        const text = await I.grabTextFrom('//div[@class="team-project-header"]');
```

Comment:
can we also move this to props?

Reply:
I have in the props. After testing forgot to use the props name